dist/init/systemd/tmpfiles.d/rkt.conf

```diff
@@ -0,0 +1,24 @@
+d /var/lib/rkt 2750 root rkt
+d /var/lib/rkt/tmp 2750 root rkt
+
+d /var/lib/rkt/cas 2770 root rkt
+d /var/lib/rkt/cas/db 2770 root rkt
+f /var/lib/rkt/cas/db/ql.db 0660 root rkt
+# the ql database uses a WAL file whose name is generated from the sha1 hash of
+# the database name
+f /var/lib/rkt/cas/db/.34a8b4c1ad933745146fdbfef3073706ee571625 0660 root rkt
+d /var/lib/rkt/cas/imagelocks 2770 root rkt
+d /var/lib/rkt/cas/imageManifest 2770 root rkt
+d /var/lib/rkt/cas/blob 2770 root rkt
+d /var/lib/rkt/cas/tmp 2770 root rkt
+d /var/lib/rkt/cas/tree 2700 root rkt
+d /var/lib/rkt/cas/treestorelocks 2700 root rkt
+d /var/lib/rkt/cas/locks 2750 root rkt
+
+d /var/lib/rkt/pods 2750 root rkt
+d /var/lib/rkt/pods/embryo 2750 root rkt
+d /var/lib/rkt/pods/prepare 2750 root rkt
+d /var/lib/rkt/pods/prepared 2750 root rkt
+d /var/lib/rkt/pods/run 2750 root rkt
+d /var/lib/rkt/pods/exited-garbage 2750 root rkt
+d /var/lib/rkt/pods/garbage 2750 root rkt
```

Comment:
- The link "directory list" in [Ownership and permissions of rkt directories](https://github.com/coreos/rkt/blob/master/Documentation/packaging.md#ownership-and-permissions-of-rkt-directories) could be updated to this `tmpfiles.d/rkt.conf` file.

- You can add a comment in [install.go](https://github.com/coreos/rkt/blob/master/rkt/install.go#L44) to say "please keep in sync with dist/init/systemd/tmpfiles.d/rkt.conf"